force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -237,12 +278,14 @@ public with sharing class FlowControlLoopsCollections {
      * @return A Set of unique strings.
      */
     public static Set<String> createStringSet(List<String> inputList) {
+        
         // Create a new Set of strings
         Set<String> stringSet = new Set<String>();
 
         //add strings to the set
+        stringSet.addAll(inputList);
```

Comment:
This way works, but you can also put the List in the constructor of the Set. 
`Set<String> stringSet = new Set<String>(inputList);`